framework/PostProcessors/ParetoFrontierPostProcessor.py

```diff
@@ -44,8 +44,10 @@ class cls.
         specifying input of cls.
     """
     inputSpecification = super(ParetoFrontier, cls).getInputSpecification()
-    inputSpecification.addSub(InputData.parameterInputFactory('costID' , contentType=InputTypes.StringType))
-    inputSpecification.addSub(InputData.parameterInputFactory('valueID', contentType=InputTypes.StringType))
+    inputSpecification.addSub(InputData.parameterInputFactory('costID' ,    contentType=InputTypes.StringType))
+    inputSpecification.addSub(InputData.parameterInputFactory('valueID',    contentType=InputTypes.StringType))
+    inputSpecification.addSub(InputData.parameterInputFactory('costLimit' , contentType=InputTypes.StringType))
+    inputSpecification.addSub(InputData.parameterInputFactory('valueLimit', contentType=InputTypes.StringType))
```

Comment:
``costLimit and valueLimit`` are float type, right?

Reply:
this is correct